vscode-extensions/tls-sync-vscode-shared/eslint.config.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+const nodeTrustedToolProfile = require('local-node-rig/profiles/default/includes/eslint/flat/profile/node-trusted-tool');
+const friendlyLocalsMixin = require('local-node-rig/profiles/default/includes/eslint/flat/mixins/friendly-locals');
+
+module.exports = [
+  ...nodeTrustedToolProfile,
+  ...friendlyLocalsMixin,
+  {
+    ignores: ['out', 'dist', '**/*.d.ts']
```

Comment:
nit: these should be unnecessary, our standard ignores groups should handle this, and `out` is not a standard output folder